.github/actions/step_publish_test_results/action.yml

```diff
@@ -43,13 +43,6 @@ runs:
       check_name: "${{ inputs.testResultTitle }} [${{ steps.display_env.outputs.GITHUB_HEAD_REF }}](https://github.com/microsoft/promptflow/actions/workflows/${{ inputs.testActionFileName }}?query=branch:${{ steps.display_env.outputs.GITHUB_HEAD_REF }}++)"
       comment_title: "${{ inputs.testResultTitle }} [${{ steps.display_env.outputs.GITHUB_HEAD_REF }}](https://github.com/microsoft/promptflow/actions/workflows/${{ inputs.testActionFileName }}?query=branch:${{ steps.display_env.outputs.GITHUB_HEAD_REF }}++)"
       files: "artifacts/**/test-*.xml"
-  - name: Generate Coverage Report
```

Comment:
why delete this?

Reply:
This is not my commit, which is introduced by https://github.com/microsoft/promptflow/pull/390/commits/d2dcac918fbf48ef8861a79c78ab72cbde3fabc0

Reply:
Please @crazygao help comment.

Reply:
do not delete this part now, please merge main again.